tests/merchOrder.test.ts

```diff
@@ -84,6 +84,65 @@ describe('merch orders', () => {
       .called();
   });
 
+  test('members can only access store with a valid acm or ucsd email', async () => {
+    const conn = await DatabaseConnection.get();
+    const UCSDMember = UserFactory.fake({ credits: 10000 });
+    const ACMMember = UserFactory.fake({
+      email: 'gZhao@acmucsd.org',
+      credits: 10000,
+    });
+    const invalidMember = UserFactory.fake({
```

Comment:
Let's use a random, arbitrary email name with the @acmucsd.org extension instead of someone's name, e.g. `random@acmucsd.org'

Reply:
Also, technically the @acmucsd.org email is for ACM board members, not any ACM member, so a variable name like `ACMBoardMember` would be a better fit